apps/www/components/particles.tsx

```diff
@@ -61,11 +60,11 @@ export const Particles: React.FC<ParticlesProps> = ({
 
   useEffect(() => {
     onMouseMove();
-  }, [mousePosition.x, mousePosition.y]);
+  }, []);
 
   useEffect(() => {
     initCanvas();
-  }, [refresh]);
+  }, []);
```

Comment:
I'd suggest checking how it affected the component behaviour. smth tells me this effect re-initiates canvas on refresh, hence `refresh` might be necessary here. the same logic as in [apps/www/components/shiny-card.tsx](https://github.com/unkeyed/unkey/pull/2648/files#diff-f49b38a359ae538213770ba2ebae7e892ad175e347c471a5059a351d63a20626)